.github/workflows/static_checks.yml

```diff
@@ -71,3 +71,11 @@ jobs:
       - name: Style checks via dotnet format (dotnet_format.sh)
         run: |
           bash ./misc/scripts/dotnet_format.sh
+
+      - name: Spell checks via codespell
+        uses: codespell-project/actions-codespell@v1
+        with:
+          skip: ./.git,./bin,./thirdparty,*.gen.*,*.po,*.pot,package-lock.json,./core/string/locales.h,./DONORS.md,./misc/dist/linux/org.godotengine.Godot.desktop,./misc/scripts/codespell.sh
+          ignore_words_list: alo,ba,childs,complies,curvelinear,doubleclick,expct,fave,findn,gird,gud,inout,lod,nd,numer,ois,readded,ro,sav,statics,te,varius,varn,wan
```

Comment:
Not relevant for this PR but does it may make sense to remove `childs` from here and replace the remaining code occurences by `children` instead (one time in the future 😄)? 
As far as I can see it is only used in `dynamic_bvh`.

Reply:
That's fair, changed in 394bb0ee2bcd863aed75e1c6799c4c8138dbf269.